template/test.Dockerfile

```diff
@@ -61,7 +73,12 @@ RUN mkdir -p $IPYTHON_CONFIG_PATH/profile_default/startup
 COPY ./template/startup_scripts/* $IPYTHON_CONFIG_PATH/profile_default/startup
 
 # Setup entrypoint for local development
-WORKDIR /home/user
+WORKDIR $HOME
 COPY ./chart_data_extractor ./chart_data_extractor
 RUN pip install -e ./chart_data_extractor
+
+# Change ownership of all files to user
+RUN chown -R user:user $HOME
+
+USER user
```

Comment:
why are you doing it as last thing? If you would set the user as a first thing you probably don't need change the ownership

Reply:
the problem here is: "user" does not exist in python image
and we cannot add user before we have "sudo" dependency installed

so this is why it's the last